ui/raidboss/data/triggers/o4n.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+// O4n - Deltascape 4.0 Normal
+
+[{
+
+  zoneRegex: /Deltascape \(V4.0\)/,
+  timelineFile: 'o4n.txt',
+  triggers: [
+    {
+      id: 'O4n Doom',
+      regex: / 1A:........:(\y{Name}) gains the effect of Doom/,
+      condition: function(data) {
+        return data.canCleanse();
+      },
+      alertText: function(data, matches) {
+        return {
+          en: 'Cleanse ' + data.shortName(matches[1]),
+        };
+      },
+    },
+    {
+      id: 'O4n Standard Blizzard',
+      regex: / 14:24BB:Exdeath starts using Blizzard III/,
+      infoText: {
+        en: 'Avoid Blizzard Puddles',
```

Comment:
These are just obvious floor aoes, right?

Reply:
That's correct, I think this was a leftover from around the same time I was finishing up Dun Scaith, so I probably hadn't got the memo at that point. This trigger was functional, so I didn't even think to review it.